### PR TITLE
feat(script): docker node in 100 node test

### PR DESCRIPTION
**Pull request by ailisp**
Can create K docker node and 100-K binary node now

Test Plan
------------
I run this script for total 6 node, 4 docker node (to avoid address insufficient) and works well

### CODE REVIEW

File: pytest/tests/delete_remote_nodes.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from utils import user_name
 
 machines = gcloud.list()
-to_delete_prefix = sys.argv[1] if len(sys.argv) >= 2 else f"pytest-node-{user_name()}"
+to_delete_prefix = sys.argv[1] if len(sys.argv) >= 2 else f"pytest-node-{user_name()}-"
```
Comment (bowenwang1996) [inline pytest/tests/delete_remote_nodes.py]: is this change intended?

File: pytest/tests/stress/hundred_nodes/create_gcloud_image.py [hunk 2]
```diff
@@ -40,9 +40,11 @@
     sudo apt update
 done
 
-sudo apt install -y python pkg-config libssl-dev build-essential cmake clang llvm
+sudo apt install -y python pkg-config libssl-dev build-essential cmake clang llvm docker.io
+sudo groupadd docker
+sudo usermod -aG docker $USER
 
-curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain nightly-2019-10-04
+curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain nightly-2020-03-19
```
Comment (bowenwang1996) [inline pytest/tests/stress/hundred_nodes/create_gcloud_image.py]: can we read the rust-toolchain file so that we don't have to adjust this manually?

File: pytest/tests/delete_remote_nodes.py (hunk 1 above)
Comment (ailisp) [reply pytest/tests/delete_remote_nodes.py]: yes. when i trying delete bo-node, bowen-node was selected too :joy:

File: pytest/tests/stress/hundred_nodes/create_gcloud_image.py (hunk 2 above)
Comment (ailisp) [reply pytest/tests/stress/hundred_nodes/create_gcloud_image.py]: Yup

File: pytest/tests/delete_remote_nodes.py (hunk 1 above)
Comment (bowenwang1996) [reply pytest/tests/delete_remote_nodes.py]: unfortunate :)
